bootstrap/v2/pkg/kfapp/kustomize/kustomize.go

```diff
@@ -28,6 +28,7 @@ import (
 	kftypesv2 "github.com/kubeflow/kubeflow/bootstrap/v2/pkg/apis/apps"
 	kfdefsv2 "github.com/kubeflow/kubeflow/bootstrap/v2/pkg/apis/apps/kfdef/v1alpha1"
 	"github.com/kubeflow/kubeflow/bootstrap/v2/pkg/utils"
+	"github.com/otiai10/copy"
```

Comment:
What made you pick this package? Are we using it in any other place?

Reply:
I use this for copy directory; I just google and found it.

Reply:
Yeah I ended up running into the same problem and my Googling turned up the same.